en/upgrades/decimal-upgrade-to-2-03/2-03-changes.rst

```diff
@@ -18,69 +18,69 @@ A full diff of all schema changes in 2.03 can be found in the `IATI GitHub <http
 
-- Added new dimension element (as a child of the baseline element) (`#366 <https://github.com/IATI/IATI-Schemas/issues/366>`__) (`discussion link <https://discuss.iatistandard.org/t/results-allow-disaggregations-of-results-data-included-2-03/871>`__)
+- Added new `dimension <http://iatistandard.org/203/activity-standard/iati-activities/iati-activity/result/indicator/baseline/dimension/>`__ element (as a child of the `baseline <http://iatistandard.org/203/activity-standard/iati-activities/iati-activity/result/indicator/baseline/>`__ element) (`#366 <https://github.com/IATI/IATI-Schemas/issues/366>`__) (`discussion link <https://discuss.iatistandard.org/t/results-allow-disaggregations-of-results-data-included-2-03/871>`__)
 
 Organisations Schema
 -------------------
 
-- Updated definition: organisation-identifier element (`#361 <https://github.com/IATI/IATI-Schemas/issues/361>`__) (`discussion link <https://discuss.iatistandard.org/t/migration-of-organisationregistrationagency-codelist-to-org-id-guide-included-2-03/851>`__)
+- Updated definition: `organisation-identifier <http://iatistandard.org/203/organisation-standard/iati-organisations/iati-organisation/organisation-identifier/>`__ element (`#361 <https://github.com/IATI/IATI-Schemas/issues/361>`__) (`discussion link <https://discuss.iatistandard.org/t/migration-of-organisationregistrationagency-codelist-to-org-id-guide-included-2-03/851>`__)
 
 Activities and Organisations Schemas
 -------------------
 
 - Updated definition: all @xml:lang attributes (`#336 <https://github.com/IATI/IATI-Schemas/issues/336>`__) (`discussion link <https://discuss.iatistandard.org/t/language-recommend-use-of-iso-639-1-included-2-03/842>`__)
 
-- Updated definition: [reporting-org, participating-org, receiver-org, provider-org] @ref attribute (`#339 <https://github.com/IATI/IATI-Schemas/issues/339>`__) (`discussion link <https://discuss.iatistandard.org/t/migration-of-organisationregistrationagency-codelist-to-org-id-guide-included-2-03/851>`__)
+- Updated definition: [`reporting-org <http://iatistandard.org/203/activity-standard/iati-activities/iati-activity/reporting-org/>`__, `participating-org <http://iatistandard.org/203/activity-standard/iati-activities/iati-activity/participating-org/>`__, `receiver-org <http://iatistandard.org/203/activity-standard/iati-activities/iati-activity/transaction/receiver-org/>`__, `provider-org <http://iatistandard.org/203/activity-standard/iati-activities/iati-activity/transaction/provider-org/>`__] @ref attribute (`#339 <https://github.com/IATI/IATI-Schemas/issues/339>`__) (`discussion link <https://discuss.iatistandard.org/t/migration-of-organisationregistrationagency-codelist-to-org-id-guide-included-2-03/851>`__)
```

Comment:
If linking to each instance, there seem to be 7 rather than 4 to link to https://github.com/IATI/IATI-Schemas/issues/339

Reply:
Also, IATI/IATI-Schemas#339 doesn't mention `reporting-org` in the checklist.

Reply:
Would suggest that either all impacted elements should be listed and linked to, or simply list the element names.